.github/workflows/ci.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           - 27017:27017
 
     env:
-      MYSQL_URL: mysql://root:1234@127.0.0.1:3306/zenstruck_foundry?serverVersion=5.7
+      MYSQL_URL: mysql://root:1234@127.0.0.1:3306/zenstruck_foundry?serverVersion=5.7.42
```

Comment:
this fixes a new deprecation in doctrine
https://github.com/doctrine/dbal/blob/3.6.x/src/Driver/AbstractMySQLDriver.php#L58-L63